client/app/models/user.js

```diff
@@ -0,0 +1,5 @@
+import Model from '@ember-data/model';
+
+export default class UserModel extends Model {
+
```

Comment:
Will need attributes:
- contactId <-- camelcased here, I think
- emailaddress1

Reply:
yup! not camelcase tho